Packs/FeedMicrosoftIntune/Integrations/FeedMicrosoftIntune/FeedMicrosoftIntune.yml

```diff
@@ -0,0 +1,97 @@
+commonfields:
+  id: Microsoft Intune Feed
+  version: -1
+fromversion: 5.5.0
+name: Microsoft Intune Feed
+display: Microsoft Intune Feed
+category: Data Enrichment & Threat Intelligence
+description: Use the Microsoft Intune Feed integration to get indicators from the
+  feed.
+configuration:
+- display: Fetch indicators
+  name: feed
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Indicator Reputation
+  name: feedReputation
+  defaultvalue: Good
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+- display: Source Reliability
+  name: feedReliability
+  defaultvalue: F - Reliability cannot be judged
+  type: 15
+  required: true
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  additionalinfo: Reliability of the source providing the intelligence data
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: suddenDeath
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- display: ""
+  name: feedExpirationInterval
+  defaultvalue: "20160"
+  type: 1
+  required: false
+- display: Feed Fetch Interval
+  name: feedFetchInterval
+  defaultvalue: "30"
+  type: 19
+  required: false
+- display: The Microsoft Intune endpoint URL
+  name: url
+  defaultvalue: https://docs.microsoft.com/en-us/intune/fundamentals/intune-endpoints
+  type: 0
+  required: true
```

Comment:
```suggestion
  required: true
- additionalinfo: Supports CSV values.
  display: Tags
  name: feedTags
  required: false
  type: 0
```

Reply:
A wide-sweeping update that we are making to our feed integrations is to add a `feedTags` parameter so that when a user configures an instance of a feed integration, they can set it so that all indicators fetched from that feed integration instance are tagged with the values entered in this field.